opm/material/fluidstates/BlackOilFluidState.hpp

```diff
@@ -321,7 +321,11 @@ class BlackOilFluidState
      * exception!
      */
     Scalar internalEnergy(unsigned phaseIdx OPM_UNUSED) const
-    { return (*enthalpy_)[phaseIdx] - pressure(phaseIdx)/density(phaseIdx); }
+    {
+        //double ent_fac=0.0;
+        double ent_fac=1.0;
```

Comment:
camelCase please

Reply:
in principle this is the same but internalEnergy only forward to a stored entalpy. For eclipse type of models it is probably simpler to do the oposite.